core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryRestApplication.java

```diff
@@ -49,6 +50,13 @@ public SchemaRegistryRestApplication(Properties props) throws RestConfigExceptio
     this(new SchemaRegistryConfig(props));
   }
 
+  @Override
+  public void configureBaseApplication(
+      Configurable<?> config, Map<String, String> metricTags
+  ) {
+    super.configureBaseApplication(config, metricTags);
```

Comment:
This puzzles me. 
It's overriding the base method only to call the base method? Seems like omitting this will have the same effect.

Reply:
That was an unintended change. Thanks for catching it :)